.github/workflows/rosa-cluster-auto-delete-on-schedule.yml

```diff
@@ -0,0 +1,26 @@
+name: ROSA Scheduled Delete
+
+on:
+  schedule:
+    - cron: '0 19 * * *' # Runs every day at 7 PM UTC
+
+jobs:
+
+  delete:
+    name: Delete ROSA cluster
```

Comment:
Wonder if we would want to re-use `rosa-cluster-delete.yml` here instead? We could add an input to that workflow `Delete all`, which would delete all, then it this scheduled workflow we could run that workflow like suggested below.

Reply:
@stianst didn't fully understand your suggestion here, we could have an additional flag within `rosa-cluster-delete.yml` if you would like to `Delete all`., but I was not sure how to add multiple trigger events for both `workflow_dispatch` to do a manual run as well as `schedule` based cron run., hence had to split out the workflow. If we could do the two triggers in the same workflow., that would be cool and save us from repeating similar steps.